Sources/SymbolKit/UnifiedSymbolGraph/GraphCollector.swift

```diff
@@ -58,6 +58,11 @@ extension GraphCollector {
     ///   - inputGraph: The symbol graph to merge in.
     ///   - url: The file name where the given symbol graph is located. Used to determine whether a symbol graph
     ///     contains primary symbols or extensions.
+    ///   - forceLoading: A Boolean value, that defaults to `false`, that indicates whether the graph is merged
+    ///     even though it is a `primary` symbol graph.
+    ///
+    /// If the symbol graph is the `primary` graph, this method ignore merging the graph if it isn't already
+    /// registered in the graph collector. To force the loading into the collector, set `forceLoading` to `true`.
```

Comment:
This explanation is actually backwards 😅 Extension graphs are held until the end, so if `forceLoading` is false or unset, then instead of processing extension graphs it will simply hold them aside. This is so that "primary" graphs can be processed first, and relationships from "extension" graphs have something to point to.

```suggestion
    ///   - forceLoading: Whether or not to force processing an extension symbol graph.
    ///
    /// By default, "extension" symbol graphs are held aside and not processed immediately, to allow for
    /// the "primary" graph to be loaded first regardless of the order that symbol graphs are found.
    /// The `forceLoading` parameter is set to `true` during ``finishLoading(createOverloadGroups:)``
    /// so that extension symbol graphs are eventually loaded at the end.
```